README.md

```diff
@@ -90,6 +90,7 @@ Actions are triggered by GitHub platform events directly in a repo and run on-de
 
 #### Notifications and Messages
 
+ - [Send a Telegram Message](https://github.com/appleboy/telegram-action)
```

Comment:
@appleboy thank you for taking the time to create this PR. As per the contributing guidelines, can I please ask that you don’t add your link at the top of the section, but rather at the bottom:

https://github.com/sdras/awesome-actions/blob/master/contributing.md#adding-to-this-list

Reply:
@gep13 Done.

Reply:
@appleboy thank you for updating this!